unstructured_platform_plugins/schema/json_schema.py

```diff
@@ -90,6 +91,8 @@ def union_type_to_json_schema(t: UnionType) -> dict:
 
 def dataclass_to_json_schema(class_or_instance) -> dict:
     resp = {"type": "object"}
+    if isinstance(class_or_instance, FileData) or class_or_instance is FileData:
+        resp["is_file_data"] = True
```

Comment:
It took me sometime to grok this, what's the difference between two checks. Maybe we could simplify it to make it more obvious, eg
```
is_filedata_instance = isinstance(class_or_instance, FileData)
is_filedata_class = class_or_instance is FileData
if is_filedata_instance or is_filedata_class: 
```